From d1ecb0e9ec841e4952127232d2bb1709ba06a5c7 Mon Sep 17 00:00:00 2001
From: Mukul Sharma <mukul@codeaurora.org>
Date: Wed, 3 Oct 2018 16:00:16 +0530
Subject: [PATCH 09/16] sepolicy: Add vendor wifi prop in vendor partition
 access

Due to never allow for wifi_prop, vendor process cannot
access the system wifi_prop property. But in certain
scenario other process need it. Hence add a support of
same.

Change-Id: I6648e0e038888e79bee5f987bc584ad126924a66
CRs-fixed: 2293118
---
 common/property.te       | 3 +++
 common/property_contexts | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/common/property.te b/common/property.te
index 7881756..8ebc0a0 100644
--- a/common/property.te
+++ b/common/property.te
@@ -136,3 +136,6 @@ type mmi_prop, property_type;
 
 #imsrcsservice
 type ctl_vendor_imsrcsservice_prop, property_type;
+
+#wlan-vendor prop
+type vendor_wifi_prop, property_type;
diff --git a/common/property_contexts b/common/property_contexts
index 3625b0f..b039849 100644
--- a/common/property_contexts
+++ b/common/property_contexts
@@ -130,3 +130,6 @@ persist.vendor.crash.detect        u:object_r:crash_prop:s0
 persist.vendor.mmi.        u:object_r:mmi_prop:s0
 
 ctl.vendor.imsrcsservice   u:object_r:ctl_vendor_imsrcsservice_prop:s0
+
+#vendor-wlan
+vendor.wlan.  u:object_r:vendor_wifi_prop:s0
-- 
2.17.1

